app/assets/javascripts/discourse/app/controllers/user-private-messages.js

```diff
@@ -41,10 +41,12 @@ export default class extends Controller {
   get messagesDropdownValue() {
     let value;
 
+    const currentURL = this.router.currentURL.toLowerCase();
```

Comment:
Is there a way to get the "full current URL" such that it is no longer necessary to alter the generated URL (which has a very confusing name, `row.id`)?

Reply:
I thought about that too but Ember's router does not provide any function that returns a URL which takes into account of the `rootURL`. `row.id` is a select-kit implementation detail so unfortunately we don't have a better alternative for naming here.